opentelemetry-sdk/src/metrics/in_memory_exporter.rs

```diff
@@ -263,7 +262,6 @@ impl InMemoryMetricExporter {
     }
 }
 
-#[async_trait]
```

Comment:
Is it not necessary to keep the `export` function definition consistent with the trait i.e., to return 
`impl std::future::Future<Output = OTelSdkResult<()>> + Send?`

Reply:
Not to my knowledge. You only have to return a future in the trait definition not where you implement it.